tsconfig.json

```diff
@@ -41,7 +42,8 @@
   "include": [
     "src",
     "test",
-    "jest.config.ts"
+    "jest.config.ts",
+    "src/**/*.json",
```

Comment:
Why is this line necessary? Is this not included under `"src"`?

Reply:
This is required import .json examples at the top of file `src\helper-modules\ordnance-survey\ordnance-survey.service.test.ts`.

Looks like typescript is not including .json files for importing.